apps/deploy.html.md.erb

```diff
@@ -69,6 +69,10 @@ You can specify one of the following deployment strategies:
 * `rolling` (default): wait for each Machine to be successfully deployed before starting the update of the next one
 * `immediate`: bring all Machines down for update at once
 * `canary`: boot a single new Machine, verify its health, and then proceed with a rolling restart strategy
+* `bluegreen`: boot a new Machine alongside each running Machine in the same region, and migrate traffic to the new Machines only once they pass health checks
```

Comment:
```suggestion
* `bluegreen`: boot a new Machine alongside each running Machine in the same region, and migrate traffic to the new Machines only once all new Machines pass health checks
```

I wonder what happens if they don't?

Reply:
For all of these, they only have health checks if the user defines some, right? I wonder what happens if you don't set checks and the app crashes on the new Machines.

Reply:
maybe @gwuah can help answer these questons! 🙂

Reply:
@gwuah which health checks does it use? Is it only the top level ones?

Reply:
I'm going to merge this rather than hold it up for new info about health checks that we can add later. I think the health checks failing, as well as which health checks are used applies to all the strategies with the exception of `immediate`.

Reply:
No worries! It should be another PR anyway. :) TY!